sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -1068,16 +1068,6 @@ std::unique_ptr<CurlNetworkConnection> CurlConnectionPool::GetCurlConnection(
         + std::string(curl_easy_strerror(result)));
   }
 
-  // curl_easy_setopt(newHandle, CURLOPT_VERBOSE, 1L);
-  // Set timeout to 24h. Libcurl will fail uploading on windows if timeout is:
-  // timeout >= 25 days. Fails as soon as trying to upload any data
-  // 25 days < timeout > 1 days. Fail on huge uploads ( > 1GB)
-  if (!SetLibcurlOption(newHandle, CURLOPT_TIMEOUT, 60L * 60L * 24L, &result))
-  {
-    throw Azure::Core::Http::TransportException(
-        Details::c_DefaultFailedToGetNewConnectionTemplate + host + ". "
-        + std::string(curl_easy_strerror(result)));
-  }
```

Comment:
Unit tests?

If not, how do you test this locally to make sure removing it will work as expected?

Reply:
I ran the getting started from Storage.
Then updated  the code to upload and download 500Mb, 1Gb and 1.5 Gb.
Tested that no timeout exception happened as it used to happened before.

In the past, the timeout exception would occur on Windows by even downloading a few MB without this config.
It looks like at some point in the past, one of the LibCurl enhancement changes (the pool or something) fixed this issue and we no longer require this patch.

It would be great to have some nightly tests to perform the big transfer tests (more than 1 GB) and see we don't throw.

In the past, @Jinming-Hu  ran this sort of tests locally with big data sets